faq/billing.mdx

```diff
@@ -122,4 +123,54 @@ You can add a billing contact directly from the User Account section of your Sca
 
 ## How can I obtain a SEPA Mandate?
 
-Refer to the [How to add a payment method](/console/billing/how-to/add-payment-method#how-to-add-a-sepa-mandate) documentation page for a complete guide on how to add a SEPA mandate.
\ No newline at end of file
+Refer to the [How to add a payment method](/console/billing/how-to/add-payment-method#how-to-add-a-sepa-mandate) documentation page for a complete guide on how to add a SEPA mandate.
+
+## My payment verification code expired. What should I do?
+
+If your verification code expires, please [contact our support team](https://console.scaleway.com/support/tickets/create) to request a new magic code.
```

Comment:
Maybe we should still to either calling it a verification code or a magic code? I don't know which one is actually used in the console? Below we also mention "transaction code" - it could be confusing to use different names for the same thing?

Reply:
Good point. I'll update it to validation code, to match the console. Thanks, Ro!